extension/wasm_src/Cargo.toml

```diff
@@ -3,23 +3,36 @@ name = "macro_railroad_ext"
 version = "0.1.0"
 authors = ["Lukas Lueg <lukas.lueg@gmail.com>"]
 build = "build.rs"
-edition = "2018"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
 
 [dependencies]
+wasm-bindgen = "0.2"
 railroad = "0.1"
 macro_railroad = "0.1"
-stdweb = "0.4"
-syn = { version = "1.0", default_features=false }
-rand = { version = "0.7", features=["stdweb"] }
 
-[lib]
-crate-type = ["cdylib"]
+# The `console_error_panic_hook` crate provides better debugging of panics by
+# logging them with `console.error`. This is great for development, but requires
+# all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
+# code size when deploying.
+console_error_panic_hook = { version = "0.1.6", optional = true }
+
+# `wee_alloc` is a tiny allocator for wasm that is only ~1K in code size
+# compared to the default allocator's ~10K. It is slower than the default
+# allocator, however.
+#
+# Unfortunately, `wee_alloc` requires nightly Rust when targeting wasm for now.
+wee_alloc = { version = "0.4.5", optional = true }
```

Comment:
Same as above, we don't actually care all that much about code-size, so this can be removed?!

Reply:
Ok, removed.